packages/terra-functional-testing/src/services/wdio-terra-service.js

```diff
@@ -41,14 +78,23 @@ class TerraService {
    * @param {Object} config wdio configuration object
    */
   async onPrepare(config) {
+    if (!this.serviceOptions.useRemoteReferenceScreenshots) {
+      return;
+    }
+
+    const prBaseBranch = await this.getPrBaseBranchRef();
```

Comment:
This makes sense when the workflow is for a PR build. However, for local development where I don't have a base branch from a PR, the expectation is that the user would need to set the [buildBranch](https://github.com/cerner/terra-toolkit/blob/%40cerner/terra-functional-testing%403.2.1/packages/terra-functional-testing/src/terra-cli/wdio/index.js#L43) CLI option to specify which branch to use to download the screenshots. So when running WDIO locally to download the screenshots, `getPrBaseBranchRef` would fail because we don't have a `this.serviceOptions.issueNumber` to use to get the `res.data.base.ref`.

Perhaps we need to flex the logic to call `getPrBaseBranchRef` to get the branch name only for PR build. Otherwise, it should just use `this.serviceOptions.buildBranch`.

Reply:
We decided that for now we'll add a new CLI option like `--prBaseBranch` that takes priority over the PR's base branch. Local devs can pass in that arg.

Reply:
Also: we need to doc that you must provide the remote credentials if you opt into remote screenshots!

Reply:
We decided to say in the doc that BuildBranch always refers to the head branch in github PR language.